db/health/migrate/20240726193601_set_import_config_type.rb

```diff
@@ -0,0 +1,12 @@
+class SetImportConfigType < ActiveRecord::Migration[7.0]
+  def change
+    Health::ImportConfig.all.each do |config|
+      case config.kind
+      when 'medicaid_hmis_exchange'
```

Comment:
* `Health::ImportConfig` should get an `active` column, `default false`.
* `Health::ImportConfig.active` scope added
* Where we call `Health::ImportConfig` make sure we're calling `.active` (see `drivers/medicaid_hmis_interchange/app/jobs/medicaid_hmis_interchange`)
* Migration to convert where `kind: disabled_medicaid_hmis_exchange` to rename `kind: medicaid_hmis_exchange`, make `active: false` (and update `type: 'Health::ImportConfigSsh'`